<?xml version="1.0" encoding="UTF-8"?>
<!DOCTYPE suite SYSTEM "https://testng.org/testng-1.0.dtd">
<suite name="Failed suite [Suite]" guice-stage="DEVELOPMENT">
  <test thread-count="5" name="Test(failed)">
    <classes>
      <class name="com.medicalapp.testcase.TC_001_AddNewStockDetails">
        <methods>
          <include name="AppOnFinish"/>
          <include name="setUp"/>
          <include name="AddStockTestcase" invocation-numbers="0"/>
        </methods>
      </class> <!-- com.medicalapp.testcase.TC_001_AddNewStockDetails -->
      <class name="com.medicalapp.testcase.TC_002_UpdateStock">
        <methods>
          <include name="AppOnFinish"/>
          <include name="setUp"/>
          <include name="PurchaseOrder" invocation-numbers="0"/>
        </methods>
      </class> <!-- com.medicalapp.testcase.TC_002_UpdateStock -->
      <class name="com.medicalapp.testcase.TC_004_UpdateCustomer">
        <methods>
          <include name="AppOnFinish"/>
          <include name="setUp"/>
          <include name="UpdateCustomer_Test" invocation-numbers="0"/>
        </methods>
      </class> <!-- com.medicalapp.testcase.TC_004_UpdateCustomer -->
    </classes>
  </test> <!-- Test(failed) -->
</suite> <!-- Failed suite [Suite] -->
